supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/PipelinesApiJobPaths.scala

```diff
@@ -14,7 +14,7 @@ object PipelinesApiJobPaths {
   val GcsDelocalizationScriptName = "gcs_delocalization.sh"
 }
 
-final case class PipelinesApiJobPaths(override val workflowPaths: PipelinesApiWorkflowPaths, jobKey: BackendJobDescriptorKey, override val isCallCacheCopyAttempt: Boolean = false) extends JobPaths {
+case class PipelinesApiJobPaths(override val workflowPaths: PipelinesApiWorkflowPaths, jobKey: BackendJobDescriptorKey, override val isCallCacheCopyAttempt: Boolean = false) extends JobPaths {
```

Comment:
Removing `final` from `case class` should really be a last resort...

* Are you sure this is necessary?
* Are you sure we aren't relying on `case class` magic (eg `apply`s/`unapply`s) elsewhere which this might break?

Reply:
Yes this was necessary, Mockito would not mock it with the `final`. I went down a lengthy rabbit hole trying to use a real `PipelinesApiJobPaths` but eventually had to give up. I can add a comment explaining the inadvisability of subclassing this in production code.